### PR TITLE
feat(movie-app): implement responsive styling, implement spinner js loader

### LINKED ISSUE #34: Make movie-app mobile responsive
**Is your feature request related to a problem? Please describe.**
Movie app on smaller screen sizes breaks layout.

**Describe the solution you'd like**
To make css mobile responsive for smaller screen sizes.

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
1. Made movie app to be responsive on smaller screen devices, user wouldn't be able to see certain screen elements on mobile, without making movieapp responsive. 
2. implement loader spinner

## Purpose
To make movie app viewable on smaller screen devices.

Closes #34

### CODE REVIEW

File: movie-app/src/App.css [hunk 1]
```diff
@@ -44,7 +51,9 @@ body, html {
   left: 0px;
   top: 0px;
   /* Full height */
-  min-height: 110%;
+  min-height: 150vh;
+  height: 100%;
+  max-height: 110%;
```
Comment (JorgeLVilla) [inline movie-app/src/App.css:56]: Cool use of percentages!

File: movie-app/src/App.css [hunk 2]
```diff
@@ -439,7 +441,135 @@ nav ul li {
...
+    width: 700px;
+  }
+
+}
+
+@media screen and (max-width: 414px) {
+
+  .App {
+
+  }
```
Comment (JorgeLVilla) [inline movie-app/src/App.css:501]: If this not needed, consider deleting please.

**Review by JayDampitan — APPROVED** [review]
Great responsive design! Amazing job!

**Review by JorgeLVilla — APPROVED** [review]
Looks great to me!

File: movie-app/src/App.css (hunk 2 above)
Comment (dannysanchez559) [reply movie-app/src/App.css:501]: good catch

**Pull request merged**